docs/reference/ilm/index-rollover.asciidoc

```diff
@@ -42,7 +42,7 @@ On each rollover, the new index becomes the write index.
 [[ilm-automatic-rollover]]
 === Automatic rollover
 
-{ilm-init} enables you to automatically roll over to a new index based
+{ilm-init} and {dlm-init} (in preview:[]]) enables you to automatically roll over to a new index based
```

Comment:
Should now be `enable you` without the `s`

Reply:
Sharp!!! Thanks!